### PR TITLE
improve exhaustive transformer test

### PR DESCRIPTION
**Pull request by MichaelBelousov**
- correct gathering of properties to include base properties in assertIdentityTransformation
- add an option to compare element geometry
- fix more jsonProperties remaps

### CODE REVIEW

**Comment by MichaelBelousov** [comment]
going to rebase out Deven's contributions since I made this branch off a transformation he was testing.

File: core/transformer/src/test/IModelTransformerUtils.ts [hunk 1]
```diff
@@ -294,34 +371,45 @@ export async function assertIdentityTransformation(
...
       }
+      if (sourceElem instanceof SpatialViewDefinition) {
+        const viewProps = expectedSourceElemJsonProps.viewDetails as ViewDetails3dProps | undefined;
+        if (viewProps && viewProps.acs)
+          viewProps.acs = remapElem(viewProps.acs);
+      }
       // END jsonProperties TRANSFORMATION EXCEPTIONS
-      const _eq = deepEqualWithFpTolerance( // kept for conditional breakpoints
+      const _eq = deepEqualWithFpTolerance(
+        // kept for conditional breakpoints
```
Comment (MichaelBelousov) [inline core/transformer/src/test/IModelTransformerUtils.ts:405]: move to above... damn prettier

**Review by ColinKerr — APPROVED** [review]

**Comment by MichaelBelousov** [comment]
got integration test model opening failures, going to add an arbitrary change to force retry

**Comment by MichaelBelousov** [comment]
@mergify update

**Comment by MichaelBelousov** [comment]
@mergify update

**Pull request merged**